yarn-project/aztec-rpc/src/aztec_rpc_server/aztec_rpc_server.ts

```diff
@@ -213,7 +213,7 @@ export class AztecRPCServer implements AztecRPCClient {
       contractAddressSalt,
       portalContract,
     );
-    const txContext = new TxContext(false, false, true, contractDeploymentData);
+    const txContext = new TxContext(false, false, true, contractDeploymentData, Fr.ZERO, Fr.ZERO);
```

Comment:
is this from where the kernel circuit will add these values into its public inputs? If so, the chainID and version should be read from somewhere (like a config value), rather than us hard-coding no? Otherwise for every upgrade we might have to edit it here?

Reply:
See the general comment.